### PR TITLE
IDE-2151 import liferay workspace wizard

### PR DESCRIPTION
**Pull request by AndyWu2015**

### CODE REVIEW

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java [hunk 1]
```diff
@@ -55,4 +70,80 @@ public static Job importGradleProject( File dir , IProgressMonitor monitor) thro
 
         return synchronizeGradleProjectJob;
     }
+
+    public static Job executeTask(final IProject project , final String task)
```
Comment (gamerson) [inline gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java]: Need to pass progress monitors into this method.

Comment (gamerson) [inline gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java]: Lets make this method synchroniuous and not create a job but directly execute the task.  The calling method can add a job if it needs to.

File: tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceImportOpMethods.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+            retval = Status.createErrorStatus( msg, e );
+        }
+
+        return retval;
+    }
+
+    private static void addPortalRuntimeAndServer( String location, IProgressMonitor monitor ) throws CoreException
+    {
+        SimpleDateFormat formator = new SimpleDateFormat( "yyyyMMddHHmmss" );
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/workspace/LiferayWorkspaceImportOpMethods.java]: Lets make the name of the server configurable by the user in the wizard and the default name of the server property will be "workspace directory name" server.

File: tools/plugins/com.liferay.ide.project.ui/plugin.xml [hunk 3]
```diff
@@ -226,6 +226,16 @@
             %import.binary.plugins.wizard.description
          </description>
       </wizard>
+      <wizard
+            category="com.liferay.ide.projects.import"
+            class="com.liferay.ide.project.ui.workspace.LiferayWorkspaceImportWizard"
+            icon="icons/e16/import.png"
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/plugin.xml]: Lets just use the same icon for new "liferay workspace" wizard for the input.  No need to use the other "import" icon.

**Comment by gamerson** [comment]
Please resend, thanks! <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java (hunk 1 above)
Comment (AndyWu2015) [inline gradle/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/GradleUtil.java]: sure ! sorry , I was stupid then .

**Pull request closed without merging**